public/assets/sass/pages/resources/important-links.scss

```diff
@@ -0,0 +1,54 @@
+@import "../../util/_sass-mixins";
+@import "../../util/variables";
+
+section {
+    max-width: var(--content-width);
+    margin: auto;
+    text-align: center;
+
+    .list {
+        list-style-type: none;
+        padding-left: 0;
+    }
+
+        .list-item-content {
+            border: 2px solid var(--mathsoc-pink-2);
+            border-radius: 3em;
+            padding: 1em;
+            margin: 1em;
+        }
+
+        .list-item-content:hover {
+            background-color: var(--mathsoc-pink-2);
+            color: white;
+            transition: all .25s ease-in;
+        }
+
+        .list-item-content:hover .list-item-title {
+            color: white;
+            transition: .25s ease-in;
+        }
+
+        .list-item-content:hover .list-item-icon {
+            filter: invert(100%);
+            transition: .25s ease-in;
+        }
+    
+            .list-item-link {
+                text-decoration: none;
+                display: flex;
+                align-items: center;
+                justify-content: center;
+            }
+
+                .list-item-icon {
+                    height: 100%;
+                    width: auto;
+                    padding-right: 1em;
+                }
+                
+                .list-item-title {
+                    margin: 0;
+                    font-weight: 400;
+                }
+}
```

Comment:
```suggestion
            .list-item-link {
                text-decoration: none;
                display: flex;
                align-items: center;
                justify-content: center;
            

                .list-item-icon {
                    height: 100%;
                    width: auto;
                    padding-right: 1em;
                }
                
                .list-item-title {
                    margin: 0;
                    font-weight: 400;
                }
            }
```

If you're trying to encapsulate each style block, you can enclose the child blocks within their parents like this! This will also enforce that each child is actually found in the parent

Reply:
Ohhh I see. That was what i was missing. Got it.